<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayAddSkills</class>
 <widget class="QDialog" name="DisplayAddSkills">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="lblTxtAns">
       <property name="text">
        <string>Text Answer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="txteditAns">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lblWeighatge">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>     Weightage</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leditWeightage">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnSaveQtn">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnClear">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnPreview">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLineEdit" name="leditOpt4">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblOpt4">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>Option 4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leditOpt3">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblOpt3">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>Option 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblOpt2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>Option 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblOpt1">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>Option 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leditOpt1">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leditOpt2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbtnAddQtn">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Add Question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblOptType">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>        Select Answer Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBoxOptionType">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Radio Button</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Check Box</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text Box</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnAddOptionAndWeightage">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Add Option and Weightage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnRemoveOpt">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Remove Option and Weightage</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="txteditQtn">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leditOpt1Weight">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leditOpt2Weight">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblOpt2Weight">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Option 2 Weightage</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lblOpt4Weight">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Option 4 Weightage</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblOpt3Weight">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Option 3 Weightage</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="leditOpt4Weight">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="leditOpt3Weight">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblOpt1Weight">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Option 1 Weightage</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>leditOpt1Weight</zorder>
  <zorder>leditOpt2Weight</zorder>
  <zorder>lblOpt2Weight</zorder>
  <zorder>lblOpt4Weight</zorder>
  <zorder>lblOpt3Weight</zorder>
  <zorder>leditOpt4Weight</zorder>
  <zorder>leditOpt3Weight</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>lblOpt1Weight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
